### PR TITLE
Quickfix: Fix Exercise Score Completion Date Sort

### PR DESCRIPTION
**Pull request by stefanwaldhauser**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

Sorting the exercise scores table by completion date did not work.

### Description

Removed the DatePipe which prevented update of the UI after sorting. Also adapted the format to a universal format that fits english and german

### Steps for Testing

1. Log in to Artemis
2. Navigate to Exercise Scores / Participation Scores
3. Check that sorting by date works correctly

Good Exercise To Test: https://artemistest.ase.in.tum.de/#/course-management/1/exercises/696/scores

### Screenshots

image

image

### CODE REVIEW

**Review by MaisiKoleni — COMMENTED** [review]
Code looks fine to me and the sorting works well, could we also fix the sorting in this PR in some other places as well? One that I found where sorting is not working properly was the student exams and exam test runs table. I don't know if the submission table has similar problems.

**Comment by stefanwaldhauser** [comment]
> Code looks fine to me and the sorting works well, could we also fix the sorting in this PR in some other places as well? One that I found where sorting is not working properly was the student exams and exam test runs table. I don't know if the submission table has similar problems.

What columns are not sorting correctly?

**Comment by MaisiKoleni** [comment]
> > Code looks fine to me and the sorting works well, could we also fix the sorting in this PR in some other places as well? One that I found where sorting is not working properly was the student exams and exam test runs table. I don't know if the submission table has similar problems.
> 
> What columns are not sorting correctly?

I think it was the start / submission date columns.

File: src/main/webapp/app/exercises/shared/participation/participation.component.html [hunk 1]
```diff
@@ -125,7 +125,8 @@ <h2>
                         </span>
                     </ng-template>
                     <ng-template ngx-datatable-cell-template let-value="value">
-                        {{ value.format('MMMM Do YYYY, h:mm:ss a') }}
+                        <!-- Todo: Had to Remove Date Pipe as UI did not update when sorting. Investigate the reason -->
```
Comment (krusche) [inline src/main/webapp/app/exercises/shared/participation/participation.component.html:128]:
```suggestion
                        <!-- TODO: Had to Remove Date Pipe as UI did not update when sorting. Investigate the reason -->
```

File: src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html [hunk 2]
```diff
@@ -157,7 +157,8 @@ <h2>
                         </span>
                     </ng-template>
                     <ng-template ngx-datatable-cell-template let-value="value">
-                        {{ value | artemisDate }}
+                        <!-- Todo: Had to Remove Date Pipe as UI did not update when sorting. Investigate the reason -->
```
Comment (krusche) [inline src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html:160]:
```suggestion
                        <!-- TODO: Had to Remove Date Pipe as UI did not update when sorting. Investigate the reason -->
```

Comment (krusche) [inline src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html:160]:
```suggestion
                        <!-- TODO: Had to Remove Date Pipe as UI did not update when sorting. Investigate the reason -->
```

**Review by krusche — APPROVED** [review]

**Pull request merged**